pkg/size/size.go

```diff
@@ -30,6 +30,13 @@ import (
 	"strings"
 )
 
+// GetAllDefaultSizes returns all the default sizes
+func GetAllDefaultSizes() []*sizev1.Size {
```

Comment:
Why not put the size definitions into a map and return that?  Caller can decide how they want to use it

Reply:
done